### PR TITLE
Pills are explosion resistant (partially reverts #15851)

### PR DESCRIPTION
**Pull request by K-Dynamic**
## About the PR
Partially reverts #15851; adds 97.5% explosion resist to pills.

## Why / Balance

Gives utility to pills as explosion-resistant drugs, making them attractive for combat.

It is also currently unintuitive to why pills and not other items (e.g. topicals and syringes) should be destroyed in explosions.

The original PR was likely intended to dissuade 200+ pill piles in older gameplay, which caused entity spam and lag. However, pills have almost disappeared for other reasons:
- Jugmed and infinite storage ChemMasters
- Moths can't eat pills, vox can't eat them safely
- Limited chems in ChemDispensers

As such, reverting is unlikely to cause pill piles to return.

## Technical details

yml changes only

`Resources/Prototypes/Entities/Objects/Specific/chemistry.yml` (adds explosion damage coefficient 0.025 to pills)

## Requirements

- [x] I have read and am following the [Pull Request and Changelog Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html).
- [x] ~I have added media to this PR or~ it does not require an ingame showcase.

## Breaking changes

**Changelog**

:cl:
- tweak: Pills are explosion resistant.

### CODE REVIEW

**Comment by BolloTea** [comment]
This is not related to this PR, but it would be great if the Spitz really could be destroyed by explosions, just like the pill bottles (not the pills themselves).

**Comment by thebadman4662** [comment]
> Does this remove the ability to have pills dissolve in liquids?

Theres only part about being destructible in commit so nope it seems.

**Comment by superjj18** [comment]
Can we make them butcherable like animal corpses so we still have the ability to extract the chemical quickly without needing a blender?

**Comment by thebadman4662** [comment]
> Can we make them butcherable like animal corpses so we still have the ability to extract the chemical quickly without needing a blender?
Yeah, would be great for something like charcoal... or just let people crush pill into container or even on floor if its empty instead of dissolving. Liquids and powders are exact same thing in the game as far as reagents go in their non-raw/pill/food from so there isn't really reason within SS14 world logic for it to not be the case other than it not being coded yet.

**Comment by BramvanZijp** [comment]
If you use the pill on a container which contains a liquid, the pill will dissolve and the contents will be put in said container, so that should not be an issue.

**Comment by Aisu9** [comment]
just make patches, this will cut down on hassle balance PR about what is destructible/what is convenient in what situation.

**Comment by SlamBamActionman** [comment]
It was brought up in discussion that it would be very weird if pills survived nukes, so instead of making them indestructible perhaps a high threshold of damage is better?

I do not think it is necessary to make the pills butcherable. There are already ways to extract pill ingredients that doesn't result in a weird puddle on the floor; grinding is a safe and pure way to do it, and dissolving the pill in liquid is a close second.

**Comment by IamVelcroboy** [comment]
Yeah, "puddles" need some sort of powder state. See: flour puddles, pill contents, etc. Very weird that they act like liquid.

**Comment by superjj18** [comment]
> I do not think it is necessary to make the pills butcherable. There are already ways to extract pill ingredients that doesn't result in a weird puddle on the floor; grinding is a safe and pure way to do it, and dissolving the pill in liquid is a close second. 

I can extract the microdose of meth from my irl adhd medication with my fingers

File: Resources/Prototypes/Entities/Objects/Specific/chemistry.yml [hunk 1]
```diff
@@ -481,12 +481,8 @@
...
       food:
         maxVol: 20
-  - type: SolutionSpiker
-    sourceSolution: food
-  - type: Extractable
-    grindableSolutionName: food
-  - type: StaticPrice
-    price: 0
+  - type: ExplosionResistance
+    damageCoefficient: 0.025
```
Comment (UbaserB) [inline Resources/Prototypes/Entities/Objects/Specific/chemistry.yml:485]: you should probably add a comment here about exactly how strong this is supposed to be

Comment (UbaserB) [reply Resources/Prototypes/Entities/Objects/Specific/chemistry.yml]: change "and not minibombs and nukes" to "but not minibombs and nukes"

**Pull request merged**